### PR TITLE
MRNN 57 - release webhook event views

### PR DESCRIPTION
**Pull request by zach-bob**
This PR implements [MRNN 57](https://mojotech.atlassian.net/jira/software/projects/MRNN/boards/22?selectedIssue=MRNN-57) by ensuring that all payloads of all webhook events are being persisted to the database as a jsonb object and adding a view to the webhook events table. 
The view uses the data inside of the jsonb object field to search for events that match the given release id and if that id has a published event. This allows us to handle release edit events for releases that have already been published.

### CODE REVIEW

File: lib/release_notes_bot/release_tags.ex [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          raw_payload -> 'release' -> 'tag_name' as tag_name,
+          raw_payload ->> 'action' as event_type
+        FROM
+          webhook_events
+      )
+      SELECT
+        tag_name
+      FROM we
+      WHERE
+        event_type = 'released' and release_id = '#{release_tag_id}';
```
Comment (chewnoill) [inline lib/release_notes_bot/release_tags.ex:27]: is this doing string interpolation for this param? Is there a way to add this as a SQL query param?

Avoid SQL injection attacks

Comment (zach-bob) [reply lib/release_notes_bot/release_tags.ex:27]: Right now the path towards this is unclear to me. I can look into this more when I return. For now, this does work. If we need this merged, we can merge it in and make a ticket to handle this sql injection problem.

File: lib/release_notes_bot/release_tags.ex [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      [] ->
+        false
+    end
+  end
+
+  defp check_for_published_events(repo_id) do
+    Repo.all(
+      from we in WebhookEvent,
+        where: we.repository_id == ^repo_id and fragment("raw_payload->>'action' = 'released'"),
+        select: fragment("raw_payload->'release'->>'id'")
```
Comment (chewnoill) [inline lib/release_notes_bot/release_tags.ex:31]: :heart:

**Review by chewnoill — APPROVED** [review]
🚀

**Pull request merged**